openfisca_core/tools/memory.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+
+"""
+A module to investigate openfisca memory usage
```

Comment:
Please add one or two lines explaining how to use it, when it can be useful (and where the alternatives such as profilers fail if you know of any).